src/actionTypes.js

```diff
@@ -341,35 +341,37 @@ type EventMessageDeleteAction = {|
   type: typeof EVENT_MESSAGE_DELETE,
   messageIds: number[],
 |};
+type TopicLinks = $ReadOnlyArray<{| +text: string, +url: string |}> | $ReadOnlyArray<string>;
+// This is current to feature level 109.
 type EventUpdateMessageAction = {|
   ...ServerEvent,
   type: typeof EVENT_UPDATE_MESSAGE,
-  edit_timestamp: number,
+  user_id: UserId,
   message_id: number,
-  // TODO is it really right that just one of the orig_* is optional?
-  orig_content: string,
-
-  // TODO: The doc for this field isn't yet correct; it turns out that
```

Comment:
The second TODO is resolved with Tim's clarifications in in
    8295bb918.

Let's clarify what repo that commit ID is in, like `zulip/zulip@8295bb918`.